### ISSUE TITLE
Port the scotch api example in kernel_gateway to R and Julia

### ISSUE THREAD

**Issue by parente** [jupyter/kernel_gateway_demos#4]
Create a new subfolder here with a Dockerfile that derives from jupyter-incubator/datascience-notebook but runs the kernel gateway instead of the notebook server. Then create two notebooks, one in R and one in Julia, that port the scotch recommendation API from the Python example provided in the kernel_gateway repo.

Include a README.md that explains how to run the demo(s) as well.

**Comment by poplav** [comment]
Pushed an initial port of the [Notebook to R](https://github.com/poplav/kernel_gateway_demos/blob/issue-4-port-scotch-api/scotch_demo/notebooks/scotch_api_r.ipynb) with the Dockerfile, Makefile, and Readme [here](https://github.com/poplav/kernel_gateway_demos/tree/issue-4-port-scotch-api/scotch_demo).  Will be starting the port to Julia now.

**Comment by poplav** [comment]
The R Notebook API is complete without issues.  

Regarding the Julia API I had to change the demo API container to run as user jovyan and open permissions for the /srv/notebooks directory where the kernel gateway reads the seeded notebook from.  There may be an issue with how Julia is installed in the [datascience-notebook Dockerfile](https://github.com/jupyter/docker-stacks/blob/master/datascience-notebook/Dockerfile#L88).  It looks like R is installed globally, but Julia was treated differently.  When running bash in the container and Julia as root, Pkg.status() results in no installed packages, but as user jovyan all packages are there as expected.

Additionally with the Julia API I opened an issue regarding reading the request code in the kernel gateway.  The Julia REPL treats `'` differently when interpreting strings in comparison to R/Python.  The issue is open [here](https://github.com/jupyter-incubator/kernel_gateway/issues/57).
